src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -205,7 +206,7 @@ namespace AppInstaller::CLI::Workflow
             context.Reporter.Warn() << "Installation abandoned" << std::endl;
             AICLI_TERMINATE_CONTEXT(E_ABORT);
         }
-        else if (installResult.value() != 0)
+        else if (installResult.value() != 0 && (std::find(successCodes.begin(), successCodes.end(), static_cast<int>(installResult.value())) == successCodes.end()))

```

Comment:
>successCodes [](start = 58, length = 12)

I think we are making a problem for ourselves later if we simply add this support without a healthy set of defaults in place.  If we take this as is, this is the problem I imagine:
1. Installer always return some value stating "reboot required for full functionality"
2. Manifest author puts this value in success codes
3. We add defaults later that make this value and others success for installation
4. Installer starts returning one of the other values in some cases (maybe reboot required for any functionality)
5. We fail the install because it isn't one of the values in the list
Then do we need to start treating the values in the manifest as additional ones, beyond 0 and our defaults?  How could they indicate to us that our defaults are incorrect; that one of them means a failure?

If the manifest called these `AdditionalSuccessCodes`, we would probably be fine to treat 0 and any values we wanted (on a per-InstallerType basis) as success.  But if it calls them `InstallerSuccessCodes`, it seems like this is the list, full stop.